tensorflow-frameworks/tensorflow-utils/src/main/java/org/tensorflow/utils/Pair.java

```diff
@@ -0,0 +1,33 @@
+package org.tensorflow.utils;
+
+public class Pair<T> extends Tuple2<T, T> {

```

Comment:
Do we need both `Pair` and `Tuple2`? Or indeed either of them?

Reply:
I had the same [comment](https://github.com/tensorflow/java/pull/30/files#r390716446), I think Dhruv can safely discard them

Reply:
Yup I think for now we can remove this; I originally used a `Pair` instead of `OneShotIterator`. I always miss tuples in Java :) but the Java pattern of creating bundler classes instead is pretty useful.